src/pages/eventdetails.js

```diff
@@ -1,18 +1,16 @@
-import EventCard from "../components/EventCard";
+import Events from "../components/Events";
 import "./eventdetails.css";
 
 import DummyEvents from "../api/DummyEvents";
-console.log(DummyEvents["Monday"][0]);
+
 export default function EventDetails() {
   return (
     <div className="container">
       <div className={"mx-[5rem] min-w-full"}>
         <h1>Event Details</h1>
       </div>
       <div className={"ml-[10rem] min-w-fit"}>
-        <EventCard event={DummyEvents["Monday"][0]} />
-        <hr className="my-3" />
-        <EventCard event={DummyEvents["Monday"][0]} />
+        <Events events={DummyEvents} />
```

Comment:
Looks pretty close, the main thing that needs to be changed it that it fills the whole container width. It should not fill the screen width necessarily, but at least the container width. You can also remove the event details header. Otherwise fantastic work.